gradle/libs.versions.toml

```diff
@@ -550,6 +553,7 @@ moko-permission-compose = { group = "dev.icerock.moko", name = "permissions-comp
 
 window-size = { group = "dev.chrisbanes.material3", name = "material3-window-size-class-multiplatform", version.ref = "windowsSizeClass" }
 androidx-material3-android = { group = "androidx.compose.material3", name = "material3-android", version.ref = "material3Android" }
+androidx-ui-android = { group = "androidx.compose.ui", name = "ui-android", version.ref = "uiAndroid" }
```

Comment:
this library is already defined in line `262` of `version-catalog`. and also there is no need for this to use remove it. also remove the newly added `kotlinx-serialization-json-v163` in line `500`